### PR TITLE
Updates for TileDB-Py 0.7.2

### PR DESCRIPTION
**Pull request by ihnorton**
Bump to TileDB Embedded 2.1.3, as well as additional fixes and improvements.

### CODE REVIEW

File: tiledb/core.cc [hunk 1]
```diff
@@ -64,6 +65,31 @@ bool config_has_key(tiledb::Config config, std::string key) {
   return true;
 }
 
+void check_arr(py::object py_arr) {
+  ptrdiff_t arr_p = py::cast<ptrdiff_t>(py_arr);
+  ArrowArray* array = (ArrowArray*)arr_p;
+  std::cout << "addr: " << array << std::endl;
```
Comment (Shelnutt2) [inline tiledb/core.cc:71]: Is this std::cout intentional? I think this is a debug function?

File: tiledb/core.cc [hunk 2]
```diff
@@ -816,6 +890,47 @@ class PyQuery {
     return results;
   }
 
+  std::unique_ptr<PAPair> buffer_to_pa(std::string name) {
+    if (query_->query_status() != tiledb::Query::Status::COMPLETE)
+      TPY_ERROR_LOC("Cannot convert buffers unless Query is complete");
+
+    tiledb::arrow::ArrowAdapter adapter(query_);
+    std::unique_ptr<PAPair> pa_pair(new PAPair()); // = std::make_shared<PAPair>();
```
Comment (Shelnutt2) [inline tiledb/core.cc:898]: Remove comment?

**Review by Shelnutt2 — APPROVED** [review]
Two minor comments, otherwise lgtm

File: tiledb/core.cc (hunk 1 above)
Comment (ihnorton) [reply tiledb/core.cc:71]: yes, will remove

**Pull request merged**